front/app/components/IdeaCard/Footer/ProposalFooter.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { Box, colors, Icon, Text } from '@citizenlab/cl2-component-library';
+import { useTheme } from 'styled-components';
+
+import { IIdeaData } from 'api/ideas/types';
+
+import StatusBadge from 'components/StatusBadge';
+import ProgressBar from 'components/UI/ProgressBar';
+
+import { ScreenReaderOnly } from 'utils/a11y';
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from '../messages';
+
+import CommentCount from './CommentCount';
+
+const ProposalFooter = ({
+  reactionCount,
+  reactionLimit,
+  showCommentCount,
+  idea,
+  hideIdeaStatus,
+}: {
+  reactionCount: number;
+  reactionLimit?: number;
+  showCommentCount: boolean;
+  idea: IIdeaData;
+  hideIdeaStatus?: boolean;
+}) => {
+  const ideaStatusId = idea.relationships.idea_status.data?.id;
+  const theme = useTheme();
+  if (!reactionLimit) {
+    return null;
+  }
+  return (
+    <>
+      <Box display="flex" gap="16px" alignItems="center">
+        <Box flex={'1'}>
```

Comment:
Yes, just not the window but the container it's placed it. You can also not need a separate box, it depends a bit on the structure. I need a separate box in this case, because I want to place both the icon and the text inside :)